core/object/class_db.cpp

```diff
@@ -491,7 +491,7 @@ StringName ClassDB::get_compatibility_class(const StringName &p_class) {
 	return StringName();
 }
 
-Object *ClassDB::_instantiate_internal(const StringName &p_class, bool p_require_real_class) {
+Object *ClassDB::_instantiate_internal(const StringName &p_class, bool p_require_real_class, bool p_extension_owner) {
```

Comment:
The argument name `p_extension_owner` doesn't it make it very clear what it does. Maybe:

```suggestion
Object *ClassDB::_instantiate_internal(const StringName &p_class, bool p_require_real_class, bool p_skip_post_initialize) {
```